### PR TITLE
XW-2565 | fixed Polish translation in create page modal

### PR DESCRIPTION
**Pull request by weronika**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 16 issues
* MAJOR 4 major
* MINOR 9 minor
* INFO 3 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [CreatePage.php#L63](https://github.com/Wikia/app/blob/e58c11666fe4fef4e24d61e4a5290a3f8aa14977/extensions/wikia/CreatePage/CreatePage.php#L63): Remove the unused function parameters "$title, $article, $output, $request, $mediaWiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. MAJOR [CreatePage.php#L138](https://github.com/Wikia/app/blob/e58c11666fe4fef4e24d61e4a5290a3f8aa14977/extensions/wikia/CreatePage/CreatePage.php#L138): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. MAJOR [CreatePage.php#L177](https://github.com/Wikia/app/blob/e58c11666fe4fef4e24d61e4a5290a3f8aa14977/extensions/wikia/CreatePage/CreatePage.php#L177): Remove this unused "$wgCreatePageOptions" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MAJOR [CreatePage.php#L178](https://github.com/Wikia/app/blob/e58c11666fe4fef4e24d61e4a5290a3f8aa14977/extensions/wikia/CreatePage/CreatePage.php#L178): Remove this unused "$wgEnableVisualEditorUI" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MINOR [CreatePage.php#L26](https://github.com/Wikia/app/blob/e58c11666fe4fef4e24d61e4a5290a3f8aa14977/extensions/wikia/CreatePage/CreatePage.php#L26): Define a constant instead of duplicating this literal "CreatePage" 4 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [CreatePage.php#L35](https://github.com/Wikia/app/blob/e58c11666fe4fef4e24d61e4a5290a3f8aa14977/extensions/wikia/CreatePage/CreatePage.php#L35): Define a constant instead of duplicating this literal "SpecialCreatePage" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [CreatePage.php#L99](https://github.com/Wikia/app/blob/e58c11666fe4fef4e24d61e4a5290a3f8aa14977/extensions/wikia/CreatePage/CreatePage.php#L99): Remove the literal "false" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)
1. MINOR [CreatePage.php#L160](https://github.com/Wikia/app/blob/e58c11666fe4fef4e24d61e4a5290a3f8aa14977/extensions/wikia/CreatePage/CreatePage.php#L160): Define a constant instead of duplicating this literal "article" 4 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [CreatePage.php#L165](https://github.com/Wikia/app/blob/e58c11666fe4fef4e24d61e4a5290a3f8aa14977/extensions/wikia/CreatePage/CreatePage.php#L165): Define a constant instead of duplicating this literal "title" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [CreatePage.php#L171](https://github.com/Wikia/app/blob/e58c11666fe4fef4e24d61e4a5290a3f8aa14977/extensions/wikia/CreatePage/CreatePage.php#L171): Define a constant instead of duplicating this literal "application/json; charset=utf-8" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)

File: extensions/wikia/CreatePage/CreatePage.i18n.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 	'createpage_cant_edit' => 'Could not perform edit',
 	'createpage-dialog-title' => 'Add New Page',
 	'createpage-dialog-message2' => 'Write your page title here:',
-	'createpage-dialog-message3' => 'We currently have {{NUMBEROFARTICLES}} pages on {{SITENAME}}. We need to check if the name is available.',
+	'createpage-dialog-message3' => 'We currently have {{NUMBEROFARTICLES}} {{PLURAL:$1|page|pages}} on {{SITENAME}}. We need to check if the name is available.',
```
Comment (kvas-damian) [inline extensions/wikia/CreatePage/CreatePage.i18n.php]: Are we sure that by doing that we won't break all existing translations in crowding when we upload that file there?

Comment (kvas-damian) [reply extensions/wikia/CreatePage/CreatePage.i18n.php]: I think we should be consistent in using `$1` or `{{NUMBEROFARTICLES}}`

**Comment by WikiaTech** [comment]
SonarQube analysis reported 16 issues
* MAJOR 4 major
* MINOR 9 minor
* INFO 3 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [CreatePage.php#L63](https://github.com/Wikia/app/blob/51732106c2d02082a26de1219769f74db4c0de56/extensions/wikia/CreatePage/CreatePage.php#L63): Remove the unused function parameters "$title, $article, $output, $request, $mediaWiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. MAJOR [CreatePage.php#L138](https://github.com/Wikia/app/blob/51732106c2d02082a26de1219769f74db4c0de56/extensions/wikia/CreatePage/CreatePage.php#L138): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. MAJOR [CreatePage.php#L177](https://github.com/Wikia/app/blob/51732106c2d02082a26de1219769f74db4c0de56/extensions/wikia/CreatePage/CreatePage.php#L177): Remove this unused "$wgCreatePageOptions" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MAJOR [CreatePage.php#L178](https://github.com/Wikia/app/blob/51732106c2d02082a26de1219769f74db4c0de56/extensions/wikia/CreatePage/CreatePage.php#L178): Remove this unused "$wgEnableVisualEditorUI" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MINOR [CreatePage.php#L26](https://github.com/Wikia/app/blob/51732106c2d02082a26de1219769f74db4c0de56/extensions/wikia/CreatePage/CreatePage.php#L26): Define a constant instead of duplicating this literal "CreatePage" 4 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [CreatePage.php#L35](https://github.com/Wikia/app/blob/51732106c2d02082a26de1219769f74db4c0de56/extensions/wikia/CreatePage/CreatePage.php#L35): Define a constant instead of duplicating this literal "SpecialCreatePage" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [CreatePage.php#L99](https://github.com/Wikia/app/blob/51732106c2d02082a26de1219769f74db4c0de56/extensions/wikia/CreatePage/CreatePage.php#L99): Remove the literal "false" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)
1. MINOR [CreatePage.php#L160](https://github.com/Wikia/app/blob/51732106c2d02082a26de1219769f74db4c0de56/extensions/wikia/CreatePage/CreatePage.php#L160): Define a constant instead of duplicating this literal "article" 4 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [CreatePage.php#L165](https://github.com/Wikia/app/blob/51732106c2d02082a26de1219769f74db4c0de56/extensions/wikia/CreatePage/CreatePage.php#L165): Define a constant instead of duplicating this literal "title" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [CreatePage.php#L171](https://github.com/Wikia/app/blob/51732106c2d02082a26de1219769f74db4c0de56/extensions/wikia/CreatePage/CreatePage.php#L171): Define a constant instead of duplicating this literal "application/json; charset=utf-8" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 16 issues
* MAJOR 4 major
* MINOR 9 minor
* INFO 3 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [CreatePage.php#L63](https://github.com/Wikia/app/blob/62369e3e18662d25179e11e58ef847ad2259711f/extensions/wikia/CreatePage/CreatePage.php#L63): Remove the unused function parameters "$title, $article, $output, $request, $mediaWiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. MAJOR [CreatePage.php#L138](https://github.com/Wikia/app/blob/62369e3e18662d25179e11e58ef847ad2259711f/extensions/wikia/CreatePage/CreatePage.php#L138): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. MAJOR [CreatePage.php#L177](https://github.com/Wikia/app/blob/62369e3e18662d25179e11e58ef847ad2259711f/extensions/wikia/CreatePage/CreatePage.php#L177): Remove this unused "$wgCreatePageOptions" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MAJOR [CreatePage.php#L178](https://github.com/Wikia/app/blob/62369e3e18662d25179e11e58ef847ad2259711f/extensions/wikia/CreatePage/CreatePage.php#L178): Remove this unused "$wgEnableVisualEditorUI" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MINOR [CreatePage.php#L26](https://github.com/Wikia/app/blob/62369e3e18662d25179e11e58ef847ad2259711f/extensions/wikia/CreatePage/CreatePage.php#L26): Define a constant instead of duplicating this literal "CreatePage" 4 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [CreatePage.php#L35](https://github.com/Wikia/app/blob/62369e3e18662d25179e11e58ef847ad2259711f/extensions/wikia/CreatePage/CreatePage.php#L35): Define a constant instead of duplicating this literal "SpecialCreatePage" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [CreatePage.php#L99](https://github.com/Wikia/app/blob/62369e3e18662d25179e11e58ef847ad2259711f/extensions/wikia/CreatePage/CreatePage.php#L99): Remove the literal "false" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)
1. MINOR [CreatePage.php#L160](https://github.com/Wikia/app/blob/62369e3e18662d25179e11e58ef847ad2259711f/extensions/wikia/CreatePage/CreatePage.php#L160): Define a constant instead of duplicating this literal "article" 4 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [CreatePage.php#L165](https://github.com/Wikia/app/blob/62369e3e18662d25179e11e58ef847ad2259711f/extensions/wikia/CreatePage/CreatePage.php#L165): Define a constant instead of duplicating this literal "title" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [CreatePage.php#L171](https://github.com/Wikia/app/blob/62369e3e18662d25179e11e58ef847ad2259711f/extensions/wikia/CreatePage/CreatePage.php#L171): Define a constant instead of duplicating this literal "application/json; charset=utf-8" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)

File: extensions/wikia/CreatePage/CreatePage.i18n.php (hunk 1 above)
Comment (weronika) [reply extensions/wikia/CreatePage/CreatePage.i18n.php]: You're right - uploading them to crowdin removed previous translations. I filled them with their previous versions (which disregarded plural forms) and issued a ticket for LOC team asking them to update this string's translations.

**Pull request merged**